cloudinit/net/network_state.py

```diff
@@ -840,7 +840,7 @@ def _handle_bond_bridge(self, command, cmd_type=None):
             # We accept both spellings (as netplan does).  LP: #1756701
             # Normalize internally to the new spelling:
             params = item_params.get("parameters", {})
-            grat_value = params.pop("gratuitious-arp", None)
+            grat_value = params.pop("gratuitous-arp", None)
```

Comment:
either this will break stuff, or it was never used

Reply:
Here they say both versions work
https://bugs.launchpad.net/netplan/+bug/1756701